packages/build/tests/plugins/tests.js

```diff
@@ -1056,41 +1056,83 @@ test('Does not pin netlify.toml-only plugin versions if there are no site ID', a
   const nodePath = getNodePath('9.0.0')
-  await runFixture(t, 'engines', { flags: { nodePath } })
+  await runFixture(t, 'engines', {
+    flags: { nodePath, featureFlags: { buildbot_build_plugins_system_node_version: false } },
+  })
 })
 
 test('Validate --node-path', async (t) => {
-  await runFixture(t, 'node_version_simple', { flags: { nodePath: '/doesNotExist' } })
+  await runFixture(t, 'node_version_simple', {
+    flags: { nodePath: '/doesNotExist', featureFlags: { buildbot_build_plugins_system_node_version: false } },
+  })
 })
 
-test('Provided --node-path version produces a node version warning if the build is being run in buildbot with local plugins', async (t) => {
+test('Provided --node-path version produces a node version warning if the build is being run in buildbot with local plugins and the plugins_system_node_version ff is set to false', async (t) => {
   const nodePath = getNodePath('9.0.0')
-  await runFixture(t, 'engines', { flags: { nodePath, mode: 'buildbot' } })
+  await runFixture(t, 'engines', {
+    flags: { nodePath, mode: 'buildbot', featureFlags: { buildbot_build_plugins_system_node_version: false } },
+  })
 })
 
 test('Provided --node-path version does not produce a node version warning if the build is being run locally', async (t) => {
   const nodePath = getNodePath('9.0.0')
-  await runFixture(t, 'engines', { flags: { nodePath, mode: 'cli' } })
+  await runFixture(t, 'engines', {
+    flags: { nodePath, mode: 'cli', featureFlags: { buildbot_build_plugins_system_node_version: false } },
+  })
 })
 
 test('Provided --node-path version does not produce a node version warning if plugins were installed via the UI', async (t) => {
   const nodePath = getNodePath('8.2.0')
   const defaultConfig = { plugins: [{ package: 'netlify-plugin-test' }] }
-  await runFixture(t, 'ui', { flags: { nodePath, defaultConfig } })
+  await runFixture(t, 'ui', {
+    flags: { nodePath, defaultConfig, featureFlags: { buildbot_build_plugins_system_node_version: false } },
+  })
 })
 
+// Node 8 execution triggers inconsistent snapshots. The feature flag will only be set in production where we run Node
+// 12. @TODO remove once we remove node 8 support.
+if (!version.startsWith('v8.')) {
+  test('No --node-path version validation is performed against the plugin engines if the plugins_system_node_version feature flag is set', async (t) => {
+    const nodePath = getNodePath('9.0.0')
+    await runFixture(t, 'engines', {
+      flags: { nodePath, featureFlags: { buildbot_build_plugins_system_node_version: true } },
+    })
+  })
+
+  test('Provided --node-path version has no effect for plugin executions when the feature flag plugins_system_node_version is set', async (t) => {
+    const nodePath = getNodePath('9.0.0')
+    await runFixture(t, 'engines', {
+      flags: { nodePath, mode: 'buildbot', featureFlags: { buildbot_build_plugins_system_node_version: true } },
+    })
+  })
+}
```

Comment:
These tests trigger unrelated log lines (due to the low Node version) when executed with Node 8. Since this scenario tests setting the FF to true (which only happens in prod) we can conditionally skip these tests in the Node 8 execution.